README.md

```diff
@@ -104,7 +104,16 @@ TBD
 
 ### Local cache
 
-TBD
+Segment chunks are cached locally to improve the fetching performance; as subsequent fetch requests will most likely hit the same chunk until all records are consumed.
+
+This cache is currently implemented as:
+- In-memory
+- Disk-based
+
+#### Disk-based Chunk Cache
+
+Requires a base directory (`path` configuration). 
+This directory is not created by the plugin, so it needs to be provided before starting the broker. 
```

Comment:
I think we shouldn't put this paragraph in this section, because it's the design overview, not the usage documentation. We can either create the usage section with this paragraph and lots of TBD, or drop it for now until we start writing the documentation.

Reply:
Sure, created https://github.com/aiven/tiered-storage-for-apache-kafka/issues/312 to follow up, and dropped changes on README.md